### PR TITLE
Replace statics with consts where possible

### PR DESCRIPTION
**Pull request by mati865**
## Description

_A description of the change and what it does. If relevant (such as any change that modifies the UI), **please provide screenshots** of the change:_

Micro-optimisation that doesn't have visible impact on performance but having more `const`s is nice.
No measurable build time change, size is down to 6069024 from 6084792.

## Issue

_If applicable, what issue does this address?_

Closes: #

## Testing

_If relevant, please state how this was tested. All changes **must** be tested to work:_

_Please also indicate which platforms were tested. All platforms directly affected by the change **must** be tested:_

- [ ] _Windows_
- [ ] _macOS_
- [x] _Linux_

## Checklist

_If relevant, ensure the following have been met:_

- [x] _Areas your change affects have been linted using rustfmt (`cargo fmt`)_
- [x] _The change has been tested and doesn't appear to cause any unintended breakage_
- [ ] _Documentation has been added/updated if needed (`README.md`, help menu, etc.)_
- [ ] _The pull request passes the provided CI pipeline_
- [x] _There are no merge conflicts_

### CODE REVIEW

File: src/constants.rs [hunk 1]
```diff
@@ -23,12 +23,9 @@ pub const TABLE_GAP_HEIGHT_LIMIT: u16 = 7;
...
 // Side borders
-pub static SIDE_BORDERS: Lazy<tui::widgets::Borders> =
-    Lazy::new(|| tui::widgets::Borders::from_bits_truncate(20));
-pub static TOP_LEFT_RIGHT: Lazy<tui::widgets::Borders> =
-    Lazy::new(|| tui::widgets::Borders::from_bits_truncate(22));
-pub static BOTTOM_LEFT_RIGHT: Lazy<tui::widgets::Borders> =
-    Lazy::new(|| tui::widgets::Borders::from_bits_truncate(28));
+pub const SIDE_BORDERS: tui::widgets::Borders = tui::widgets::Borders::from_bits_truncate(20);
+pub const TOP_LEFT_RIGHT: tui::widgets::Borders = tui::widgets::Borders::from_bits_truncate(22);
+pub const BOTTOM_LEFT_RIGHT: tui::widgets::Borders = tui::widgets::Borders::from_bits_truncate(28);
```
Comment (mati865) [inline src/constants.rs:28]: These 2 are unused.

Comment (ClementTsang) [reply src/constants.rs:28]: Hm, feel free to remove them if you want in this PR.

**Comment by ClementTsang** [comment]
LGTM.  Thanks!

If you want to also remove those unused constants, feel free to.  If not, I'm happy to accept this PR as is.

**Comment by ClementTsang** [comment]
Thanks!

**Pull request merged**